docs/tutorials/control/air-quality-fleet.md

```diff
@@ -0,0 +1,827 @@
+
+1. If you're not getting sensor readings from a given machine, check the path of the USB port using the same [process by which you found the first USB path](#usb-path).
+2. If the path to your sensor on one machine is different from the one you configured in the fragment, add a fragment mod to the config of that machine to change the path without needing to remove the entire fragment.
+   Follow the [instructions to add a fragment mod](/fleet/configure-a-fleet/#use-fragment_mods) to your machine's config, using the following JSON template:
+
+   ```json {class="line-numbers linkable-line-numbers"}
+   "fragment_mods": [
+   {
+     "fragment_id": "<REPLACE WITH YOUR FRAGMENT ID>",
+     "mods": [
+       {
+         "$set": {
+           "components.PM_sensor.attributes.usb_interface": "<REPLACE WITH THE PATH TO THE SENSOR ON YOUR MACHINE>"
+         }
+       }
+     ]
+   }
+   ],
+   ```
+
+   Replace the values with your fragment ID and with the USB path you identify.
+   If you named your sensor something other than `PM_sensor`, change the sensor name in the template above.
+
+3. Repeat this process for each machine that needs a different `usb_interface` value.
+   If you have lots of machines with one `usb_interface` value, and lots of machines with a second one, you might consider duplicating the fragment, editing that value, and using that second fragment instead of the first one for the applicable machines, rather than using a fragment mod for each of the machines.
+   You have options.
+
+   {{% /expand%}}
+
+## Test data sync
+
+Next, check that data is being synced from your sensors to the cloud:
+
+1.  Open your [**DATA** page](https://app.viam.com/data).
+2.  Click the **Sensors** tab within the data page.
+3.  If you have sensor data coming from machines unrelated to this project, use the filters on the left side of the page to view data from only your air quality sensors.
+    Click the **Tags** dropdown and select the `air-quality` tag you applied to your data.
+    You can also use these filters to show the data from one of your air quality sensors at a time by typing a machine name into the **Machine name** box and clicking **Apply** in the lower-left corner.
+
+    {{<imgproc src="/tutorials/air-quality-fleet/synced-data.png" resize="x1100" declaredimensions=true alt="The sensor readings that have synced to the DATA page." style="max-width:600px" >}}
+
+Once you've confirmed that data is being collected and synced correctly, you're ready to start building a dashboard to display the data.
+If you'd like to graph your data using a Grafana dashboard, try our [Visualize Data with Grafana tutorial](/tutorials/services/visualize-data-grafana/).
+If you'd like to create your own customizable dashboard using the Viam TypeScript, continue with this tutorial.
+
+## Code your custom TypeScript dashboard
```

Comment:
So here I'd like you to view this from the business's perspective "creating an overview of all your customers"
You can also allow customers to view their own data. That's where the permissions and things like that become interesting

Reply:
I believe the way you set this up the business could input their access credentials and see all machines across all locations AND the clients could input their location credentials and see their own sensors on the dashboard. I think that's really cool and you can work that into the story I think